product.json

```diff
@@ -54,7 +54,10 @@
 		"IDERA.sqldm-performance-insights",
 		"SentryOne.plan-explorer"
 	],
+	"recommendedVisualizers" :[
```

Comment:
can we change this to recommendedVisualizerExtensions?

Reply:
@alanrenmsft  Or is there a way to have the recommendations specify a category field so this can be done generically, so we don't have to keep updating the extension management code for each new category type?  Or am I missing something here?